Sources/OTel/OTel+Bootstrap.swift

```diff
@@ -20,6 +20,94 @@ import Tracing
 // MARK: - API
 
 extension OTel {
+    /// Bootstrap observability backends with OTLP exporters.
+    ///
+    /// - Parameter configuration: Configuration for observability backends.
+    ///
+    ///   This value can be used to configure or disable the bootstrapped backends and defaults to
+    ///   `OTel.Configuration.default`, which bootstraps all backends with the default configuration defined in the
```

Comment:
```suggestion
    ///   ``OTel/Configuration/default``, which bootstraps all backends with the default configuration defined in the
```

I'm not 100% sure this is the right syntax, but would be nice to use a docc link here.